Dockerfile

```diff
@@ -1,12 +1,15 @@
 FROM mambaorg/micromamba:1.4.9-bullseye-slim
 
+USER mambauser
+
 SHELL [ "/bin/bash", "-c" ]
 
 # Use C.UTF-8 locale to avoid issues with unicode encoding
 ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
 
-COPY entrypoint.sh /entrypoint.sh
+COPY --chown=mambauser entrypoint.sh /entrypoint.sh
 RUN chmod +x /entrypoint.sh
 
-ENTRYPOINT ["/entrypoint.sh"]
+ENTRYPOINT [ "/usr/local/bin/_entrypoint.sh" ]
```

Comment:
I read your PR description, but am still fuzzy on what is happening in these two lines. Maybe a comment would help?

Reply:
I think the confusion is calling ours entrypoint as well; it really is the command we execute AFTER entrypoint, so we can just give it a logical name that describes what it does?

But, this is already way better, so we could merge this, perhaps with a note to explain the distinction between ENTRYPOINT and CMD.

I.e. we use entrypoint to set env variables, and the cmd does whatever action we wish to execute.